tests/test_gimmesomeoven.py

```diff
@@ -0,0 +1,56 @@
+from tests import ScraperTest
+
+from recipe_scrapers.gimmesomeoven import GimmeSomeOven
+
+
+class TestAllRecipesScraper(ScraperTest):
```

Comment:
that here has to be `TestGimmeSomeOven` and it's good to go

Reply:
Whoops! That's embarrassing...